ADMIN-PORT.md

```diff
@@ -0,0 +1,44 @@
+---
+RFC: 3
+Status: Proposed
+---
+
+# ADMIN PORT RFC
+
+## Abstract
+
+In Valkey.conf, Add a new config parameter: "admin-port" to allow internal administrators in a company or special users from public internet to connect to the Valkey server to execute some special commands or load specfic modules.
+
+
+## Motivation
+
+For adminstrators, they may want to execute some special commands in the server, and other general clients are not allowed to run these kinds of commands.
+We want to let adminstrator connect to this admin port to do some special things. 
+
+Beside above mentioned, we could implement the following features in the future:
+1. The connections through admin port are not limited to maxclient number, we can always guarantee they can connect to the server.
```

Comment:
I thought this was one of the main features? It seems like a good idea to me. I guess maybe instead of unlimited, maybe there is a special allowance of like 128 "admin" connections.

Reply:
Sorry @madolson this is an enhancement feature based on the admin-port added. The main feature is that the connection through admin-port could call some special command. If we have admin-port, we could have more fancy features later, including this.